crates/scheduler/Cargo.toml

```diff
@@ -6,11 +6,12 @@ rust-version.workspace = true
 license.workspace = true
 
 [features]
-default = ["kv_store", "olap"]
+default = ["kv_store", "olap","v1"]
 olap = ["storage_impl/olap", "hyperswitch_domain_models/olap"]
 kv_store = []
 email = ["external_services/email"]
 v1 = ["diesel_models/v1", "hyperswitch_domain_models/v1", "storage_impl/v1", "common_utils/v1"]
+v2 = ["diesel_models/v2", "hyperswitch_domain_models/v2", "storage_impl/v2", "common_utils/v2"]
```

Comment:
Can you update `router/Cargo.toml` to enable `scheduler` crate's `v1` or `v2` feature flags, when the corresponding features are enabled in the `router` crate?